std/stdio.d

```diff
@@ -2878,12 +2878,20 @@ $(D Range) that locks the file and allows fast writing to it.
                     return;
                 }
```

Comment:
Won't the else case (i.e., `orientation_ > 0`) have a bug, now that you made the original fallthrough code an `else` to the static if?

Reply:
Yes. Fixed

Reply:
Hmm. Can't say I'm a fan of the copy-pasta. What about inserting a return for the `isSomeString!A && C.sizeof > 1` case, and getting rid of the else after that, so that we can merge these two identical for-loops?  I feel bad for nitpicking on this level, but the code duplication seems unwarranted in this case.